src/ui/components/NetworkMonitor/RequestRow.tsx

```diff
@@ -56,7 +59,7 @@ export const RequestRow = ({
         tabIndex={0}
         data-testid="NetworkMonitor-RequestTable-RequestRow"
       >
-        <div {...row.getRowProps()}>
+        <div {...rowProps}>
```

Comment:
Fixes warning about passing the key to div

Reply:
Doesn't look like this should have changed anything either. Can you explain why you think it fixed a warning?

Reply:
Hi @bvaughn, `getRowProps` return value contains the `key` so spreading it here in div causes a warning